beta-src/src/components/ui/WDMoveControls.tsx

```diff
@@ -111,6 +117,14 @@ const WDMoveControls: React.FC<WDMoveControlsProps> = function ({
 
   const saveDisabled = ready || !save;
 
+  React.useEffect(() => {
+    if (data) {
+      dispatch(gameApiSliceActions.updateUnitsDisbanding());
+      dispatch(gameApiSliceActions.drawBuilds());
+      console.log("MVC");
+    }
+  }, [data]);
+
```

Comment:
You shouldn't need to do this, you can base the icon state of the disbands on the `ordersMeta` property in global state. There is code in gameApiSlice actions which draws arrows whenever ordersMeta is updated. When the user clicks the save button and orders meta is updated, that is when you can change the icon.